paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -277,6 +277,13 @@ public class CoreOptions implements Serializable {
                     .withDescription(
                             "Whether the write buffer can be spillable. Enabled by default when using object storage.");
 
+    public static final ConfigOption<Boolean> USE_WRITE_BUFFER =
+            key("use-write-buffer")
```

Comment:
Also document this in append-only documentation page.

Reply:
done